### PR TITLE
Increase library version from 2.7.4 to 2.7.5

### PR DESCRIPTION
**Pull request by Namiii**

### CODE REVIEW

**Comment by nuxzero** [comment]
Could you add what new updating to `CHANGELOG.md` too?

**Comment by Namiii** [comment]
@nuxzero done done

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # CHANGE LOG
 
+#2.7.5
+**REMOVED:** Removed Sinarmas virtual account as a payment source type
+
+#2.7.4
+**CHANGED:** Changed variable names for Alipay barcode payment
```
Comment (nuxzero) [inline CHANGELOG.md]: I thought we have `metadata` included in `2.7.4`
Could you add below description?
```
**NEW:** Add metadata field to creating and updating a Transfer.
**NEW:** Add metadata field to creating a Refund.
**NEW:** Add metadata field to updating a Dispute.
```

Comment (nuxzero) [reply CHANGELOG.md]: @Namiii Sorry one more 😭

Comment (Namiii) [reply CHANGELOG.md]: alright, will do

Comment (Namiii) [reply CHANGELOG.md]: 👍

**Pull request merged**